examples/jenkins-master/jenkins-master-template.json

```diff
@@ -143,7 +154,7 @@
           "sourceStrategy": {
             "from": {
               "kind": "DockerImage",
-              "name": "${JENKINS_IMAGE}"
+              "name": "${NAMESPACE}/${JENKINS_IMAGE}"
```

Comment:
this won't work, it's a docker image reference and not an imagestreamtag reference.  we should ask @mfojtik why it was done this way (probably to avoid having a dependency on the a jenkins imagestream existing) and see if it would be ok to change it to an ImageStreamTag reference instead.